src/libs/Log.js

```diff
@@ -21,7 +21,7 @@ function LogCommand(parameters) {
 
     // Note: We are forcing Log to run since it requires no authToken and should only be queued when we are offline.
     // Non-cancellable request: during logout, when requests are cancelled, we don't want to cancel any remaining logs
-    return Network.post(commandName, {...parameters, forceNetworkRequest: true, canCancel: false});
+    return Network.post(commandName, {...parameters, forceNetworkRequest: true});
```

Comment:
I think it's still true that we don't want to cancel logs.

Reply:
Commented on that below